CopilotKit/packages/runtime/src/lib/runtime/remote-lg-action.ts

```
@@ -550,7 +550,15 @@ async function streamEvents(controller: ReadableStreamDefaultController, args: E
 
     state = await client.threads.getState(threadId);
     const interrupts = state.tasks?.[0]?.interrupts;
-    nodeName = interrupts ? nodeName : Object.keys(state.metadata.writes)[0];
+
+    if (interrupts) {
+      nodeName = nodeName;
```

Biome (1.9.4)

[error] nodeName is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)